### PR TITLE
Access the Map, View and Fail Samples migrated to React Function Component

### PR DESCRIPTION
**Pull request by cemrehancavdar**
This PR includues 2 basic code sample changes on Markdown.md.
Access the Map, View sample and onFail sample migrated from Class Component to Function Component (which all other samples uses Function Component structure)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,55 +118,42 @@ export default (props) => (
...
-        this.handleFail = this.handleFail.bind(this);
-    }
-
-    render() {
-        return <WebScene className="full-screen-map" id="foobar" onFail={this.handleFail} />;
-    }
-
-    handleFail(e) {
+const MakeAScene = () => {
+    const {status, setStatus} = useState('loading');
```
Comment (gavinr) [inline README.md]:
```suggestion
    const [status, setStatus] = useState('loading');

```

**Review by gavinr — CHANGES_REQUESTED** [review]
Thanks. Overall these changes to the README samples look good, just one small tweak I noticed.

**Comment by cemrehancavdar** [comment]
Oh, i see! Commited it.
Thank you 👍

**Review by gavinr — APPROVED** [review]

**Comment by cemrehancavdar** [comment]
Can you merge it? @gavinr

**Review by tomwayson — COMMENTED** [review]
My intention was to have _both_ class and functional component examples. It seems to me like there is still value in that.

**Comment by tomwayson** [comment]
Thank you @cemrehancavdar, but I am closing this as we are now archiving this repo

**Pull request closed without merging**